config.yml.example

```diff
@@ -203,6 +203,37 @@
 #service.job_cleanup_interval: 300
 #
 #
+## ------------------------------- Service  Error Monitor ----------------------------------
```

Comment:
Should we have a `service.error_monitor.enabled: true`? So that this doesn't immediately start getting used in Native Connectors without us having a way to expose the configs there? (fine if you'd rather that be in a follow-up)

Reply:
Good point, will add this!